pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -383,14 +394,25 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
 ```json
 {
  "channels": [
-   "sms", "inapp"
+   "sms"
  ],
  "skills": [
-   "billing", "sales"
+   "billing"
+ ],
+ "messagestatus": [
+   "DELIVERED", "FAILED", "RESPONDED", "READ"
+ ],
+ "transactionids": [
+   "03cb1b10-2e46-4c10-bdc7-3d9c23654d51"
+ ],
```

Comment:
@jschatzman-lp Can we remove other things and just keep the changes related to the source field. The other changes for transaction API are already present in production.

Reply:
@lp-gdoshi I'm not sure I understand why you want to remove this, I think the dev wiki should call out that we have an option to filter on transaction ID using transaction API

Reply:
@lp-gdoshi Nevermind, I see. We need to differentiate the request body between 1.0 and 2.0 for transactions api. I will fix this.